src/chai-form-configurator.ts

```diff
@@ -0,0 +1,414 @@
+        if(!checkbox) return;
+
+        if(componentName == 'chai-name')
+        {
+          target.isCreateName = checkbox.checked;
+        }else if(componentName == 'chai-email')
+        {
+          target.isCreateEmail = checkbox.checked;
+        }else if(componentName == 'chai-address')
+        {
+          target.isCreateAddress = checkbox.checked;
+        }else if(componentName == 'chai-phone')
+        {
+          target.isCreatePhone = checkbox.checked;
+        }
+  }
+
+
+  applyChaiStyle(event: Event , cssVaraible:string , px:string)
+  {
+    console.log(event,cssVaraible);
+      const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+        const input = event.target as HTMLInputElement;
+        target.style.setProperty(cssVaraible , input.value + px);
+        
+  }
+
+
+  override render() {
+    return html`
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+<div>
+ 
+ <div class="row form-configurator-container">
+  <div class="col-sm-12">
+     <h2 class="form-configurator-header-message">Form Configurator</h2>
+  </div>
+        <div class="col-sm-6">
+            <div class="row">
+                <div class="col-sm-12">
+                    <label for="form-flex-direction">Form Flex Direction</label>
+                    <select id='form-flex-direction' class="form-control" @input="${(event:CustomEvent) => {this.applyChaiStyle(event,'--chai-form-flex-direction','')}}">
+                        <option value=''>Choose Flex Direction</option>
+                        <option value='column'>Column</option>
+                        <option value='row'>row</option>
+                    </select>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-sm-12">
+                    <label for="form-color-brand">Form Color Brand</label>
+                    <input type="color" class="form-control" id="form-color-brand" @input="${(event:CustomEvent) => {this.applyChaiStyle(event,'--chai-form-color-brand','')}}">
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-sm-12">
+                    <label for="form-color-text">Form Color Text</label>
+                    <input type="color" id="form-color-text" class="form-control"  @input="${(event:CustomEvent) => {this.applyChaiStyle(event,'--chai-form-color-text','')}}">
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-sm-12">
+
+                    <label for="form-font-size">Form Font Size</label>
+                    <input type="range" class="form-control" id="form-font-size" @input="${(event:CustomEvent) => {this.applyChaiStyle(event,'--chai-form-font-size','px')}}">
```

Comment:
None of the slider controls show the user the value that's being used. It would be helpful to see how many pixels the slider is set to.